src/main/webapp/app/overview/course-conversations/course-conversations.component.ts

```diff
@@ -33,6 +33,15 @@ const DEFAULT_CHANNEL_GROUPS: AccordionGroups = {
     hiddenChannels: { entityData: [] },
 };
 
+const CHANNEL_GROUPS_MESSAGING_DISABLED: AccordionGroups = {
```

Comment:
Instead of duplicating the code, `CHANNEL_GROUPS_MESSAGING_ENABLED` should re-use all groups from `CHANNEL_GROUPS_MESSAGING_DISABLED`.
```ts
const CHANNEL_GROUPS_MESSAGING_ENABLED: AccordionGroups = {
     ...CHANNEL_GROUPS_MESSAGING_DISABLED,
     groupChats: { entityData: [] },
     directMessages: { entityData: [] },
};
```